stm32mp1.mk

```diff
@@ -59,6 +63,11 @@ STM32MP1_DEFCONFIG_U_BOOT ?= stm32mp15_defconfig
 # is intended to platforms with an eMMC device.
 WITH_RPMB_TEST ?= n
 
+# When enabled WITH_SRAM1_PAGER_POOL makes OP-TEE pager core to use secure
+# SYSRAM and SRAM1. This switch concerns STM32MP15 based platforms only.
+# When enabled, CFG_TEE_CORE_DEBUG is also default enabled.
```

Comment:
Why should debug be related to `WITH_SRAM1_PAGER_POOL`?

Reply:
OP-TEE default config sets `CFG_TEE_CORE_DEBUG ?= y`.
plat-stm32mp1/conf.mk sets CFG_TEE_CORE_DEBUG ?= n` when `CFG_WITH_PAGER=y` to reduce pager pool size constraint. Here my intend is to restore default debug mode config since using SRAM1 for pager relaxes the pressure on pager pool size.